src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -266,4 +272,34 @@ public static Pronoun parsePronoun(String pronoun) throws ParseException {
         }
         return pronounSet;
     }
+
+    /**
+     * Replaces all placeholder labels in the given {@code HashMap<Label, T>} with default labels.
+     *
+     * @param entries The {@code HashMap<Label, T>} of key-value pairs to be given default labels.
+     * @param <T> The type of the value in the given {@code HashMap<Label, T>}.
+     * @return A {@code HashMap<Label, T>} with default labels in place of placeholders.
+     */
+    public static <T> Map<Label, T> replacePlaceholdersInMap(Map<Label, T> entries) {
```

Comment:
Is there a better place we can keep the `replacePlaceholdersInMap()` function, since the **parsing** stage is already complete? Perhaps in a `util` class?

Reply:
> Is there a better place we can keep the `replacePlaceholdersInMap()` function, since the **parsing** stage is already complete? Perhaps in a `util` class?

I agree it's not really the appropriate class, but I think none of these existing util classes are appropriate either. What do you think of creating a new util class like PlaceholderUtil or something in the logic component?